addons/crm/views/crm_lead_views.xml

```diff
@@ -139,15 +139,8 @@
                                     <button name="mail_action_blacklist_remove" class="fa fa-ban text-danger"
                                         title="This email is blacklisted for mass mailings. Click to unblacklist."
                                         type="object" context="{'default_email': email_from}" groups="base.group_user"
-                                        attrs="{'invisible': ['|', ('is_blacklisted', '=', False), ('partner_address_email', '!=', False)]}"/>
-                                    <field name="email_from"
-                                        attrs="{'invisible': [('partner_address_email', '!=', False)]}" string="Email" widget="email"/>
-                                    <button name="mail_action_blacklist_remove" class="fa fa-ban text-danger"
-                                        title="This email is blacklisted for mass mailings. Click to unblacklist."
-                                        type="object" context="{'default_email': partner_address_email}" groups="base.group_user"
-                                        attrs="{'invisible': ['|', ('partner_is_blacklisted', '=', False), ('partner_address_email', '=', False)]}"/>
-                                    <field name="partner_address_email"
-                                        attrs="{'invisible': [('partner_address_email', '=', False)]}" widget="email" string="Email"/>
+                                        attrs="{'invisible': [('is_blacklisted', '=', False)]}"/>
+                                    <field name="email_from" string="Email" widget="email"/>
```

Comment:
I don't think we should remove the blacklist option.

Reply:
@tde-banana-odoo The blacklist option is still here ;) before there were 2 blacklist button, one for the field `partner_address_email` and the other the the field `email_from`.
But, because I removed the field `partner_address_email` I also removed the blacklist button

Reply:
Ok, diff was not really clear. Solved.